extensions/adobe/experience/decisioning/proposition-interaction-detail.schema.json

```diff
@@ -35,6 +35,16 @@
             "description": "An item from the proposition's options. See 'selections' of the 'Decision Proposition Detail' mix-in."
           }
         },
```

Comment:
@jbeckert I was under the impression that we will add `experience` under `scopeDetails` similar to `activity`. As far as I remember @avmaharaj said that we might want to add additional details later.  Here as an example of what I think it should look like:
```json
{
  "xdm:activity": {
    "xdm:id": "xcore:offer-activity:ebc48132c26ccfc"
  },
  "xdm:experience": {
    "xdm:id": "some-experience-id"
  },
  "xdm:strategy": [
    {
      "xdm:algorithmID": "ts",
      "xdm:trafficType": "noncontextual"
    }
  ]
}
```

Reply:
this feedback was accepted and updates have been made